README.md

```diff
@@ -1,2 +1,76 @@
-# messaging-aws-sqs-dynamodb-php
+# vonage/messaging-aws-sqs-dynamodb-php
+
 AWS Lambda function created in PHP that once triggered, moves a message from SQS into DynamoDB.
+
+## Prerequisites
+
+* PHP 7.4 (update `serverless.yml` for other versions)
+* Composer installed [globally](https://getcomposer.org/doc/00-intro.md#globally)
+* [Node.js](https://nodejs.org/en/) and npm
+* [Serverless Framework](https://serverless.com/framework/docs/getting-started/)
+* [AWS account](https://aws.amazon.com/)
+
+## Setup Instructions
+
+Clone this repo from GitHub, and navigate into the newly created directory to proceed.
+
+### Use Composer to install dependencies
+
+This example requires the use of Composer to install dependencies and set up the autoloader.
+
+Assuming a Composer global installation. [https://getcomposer.org/doc/00-intro.md#globally](https://getcomposer.org/doc/00-intro.md#globally)
+
+```
+composer install
+```
+
+### AWS Setup
+
+You will need to create [AWS credentials](https://www.serverless.com/framework/docs/providers/aws/guide/credentials/) as indicated by `Serverless`.
+
+Also, create a new [SQS queue](https://aws.amazon.com/sqs/) using the default settings, and update `serverless.yml` and `index.php` with the SQS queue URL. Do this by updating `<your-sqs-arn>` and `<your-sqs-url>` respectively.
+
+Lastly, create a new [DynamoDB table](https://aws.amazon.com/dynamodb/) using the default settings, and update `serverless.yml` and `index.php` with that information as well. Do this by updating `<your-dynamodb-table-arn>` and `<your-dynamodb-table-name>` respectively.
+
+> Note: Ensure the primary key field name you set for the DynamoDB table matches the message ID in your SQS queue items. For this example we used `messageId`.
+
+### Update Index.php
+
+Update the `index.php` to ensure the variables in the beginning of the function get defined correctly.
+
+```php
```

Comment:
Can we throw in dotenv and an `.env.example` so that the users can configure this without having to change the source code?

Reply:
Thanks. Will add this recommendation in a refactor.

Reply:
Added environmental variable use.